README.md

```diff
@@ -98,6 +98,14 @@ The site is configured to use
 startup time while Rails loads. See the Zeus github page for
 usage instructions.
 
+Once [npm dependencies are
+installed](https://github.com/openfoodfoundation/openfoodnetwork/wiki/Karma), AngularJS tests can be run with:
+
+    rake karma:run
```

Comment:
I think it is better to have a script to take care of `RAILS_ENV`. So we just have to write `karma run` or `karma start`